sniffles/test/test_ttl_expiry_attack.py

```diff
@@ -0,0 +1,23 @@
+from unittest import *
+from sniffles.ruletrafficgenerator import *
+
+
+class TestTTLExpiryAttack(TestCase):
+
+    def ttl_expiry_scenario1(self):
+      print("hello world")
```

Comment:
Please use the same indentation as everywhere else. This block uses two spaces while all the others use four spaces.